templates/lx_test.sh

```diff
@@ -67,6 +67,9 @@ if [ "$build_type" != "$build_type_builder" ] && [ "$${userdata_status[0]}" -eq
   else
     watchmaker --version
   fi
+
+  # Test sudo is functional
+  sudo cat /etc/sudoers
```

Comment:
Looking at the error in one of the codebuild jobs, I think we can avoid the timeout by using the --non-interactive switch...

```suggestion
  sudo --non-interactive cat /etc/sudoers
```

Reply:
Oh, sudo has a --list option also, to print out the user's permissions...
```suggestion
  sudo --non-interactive --list
```

Reply:
Okay.  I'll make the change.

Reply:
I switched to something less verbose.  Hope that's cool also.